pandas/core/config.py

```diff
@@ -835,3 +837,23 @@ def is_callable(obj):
     if not callable(obj):
         raise ValueError("Value must be a callable")
     return True
+
```

Comment:
why are you moving this? this defeats the entire purpose of config_init

Reply:
The goal is to move `config` out of `core` (either to another dependency-free directory in pandas or something external like @topper-123's repo) and make it explicitly "upstream" from  `_libs`.  That way we get a much cleaner dependency structure by avoiding runtime imports inside `_libs`.